### ISSUE TITLE
.webm doesn't loop

### ISSUE THREAD

**Issue by girng** [godotengine/godot#16456]
[videobugged,doesn'tloop.zip](https://github.com/godotengine/godot/files/1700941/videobugged.doesn.tloop.zip)

Godot 3.0, latest

i checked import options, it's looped enabled.

when printing `stream_position`, i think possible floating point rounding error, doesn't trigger it to get looped again? not entirely sure.

**Comment by girng** [comment]
so this has been reported for 3days but not labeled a bug?

**Comment by cyaoeu** [comment]
You have to add this behavior yourself, for example
`	if not is_playing():
		play()`
in the videoplayer delta function.
This is not a bug.

**Comment by groud** [comment]
> so this has been reported for 3days but not labeled a bug? please stop with favoritism, all bugs are created equal.

Please let the moderator do their job, there's no favoritism here but only people that might not have the time to label all bugs that are raised every day.

Related to #6179, if not a duplicate.

**Comment by groud** [comment]
Also, no need to be rude in naming your sample project. That does not fix bugs faster.

**Comment by girng** [comment]
@cyaoeu what? this is most definitely a bug. looping is set as enabled.

@groud ty for the label, although favouritism was present (3 days of this bug gaining no attention), while all the bugs issues around it were labeled. please be fair is all i ask.

**Comment by Noshyaar** [comment]
> not trying to be rude

![image](https://user-images.githubusercontent.com/26753932/36085034-4143c90e-0ff5-11e8-9af4-dcfdd435ac48.png)

> favoritism is not nice

So is being rude.

**Comment by vnen** [comment]
There are older issues without any labels.

**Comment by girng** [comment]
@vnen understood, but when i first made this issue, it was 3 days before a label was added. and there were 3 other pages surrounding it that all had labels.

**Comment by akien-mga** [comment]
Duplicate of #6179. Looping is not implemented for VideoStream yet (and after #19937 it will stop showing the unused "loop" import option, since streams are no longer imported).
